pkgs/by-name/bu/buildstream/package.nix

```
@@ -52,7 +55,10 @@ python3Packages.buildPythonApplication rec {
       ruamel-yaml-clib
       tomlkit
       ujson
-    ]);
+    ])
+    ++ lib.optional enableBuildstreamPlugins [
```

```suggestion
    ++ lib.optionals enableBuildstreamPlugins [
```

If you're using `lib.optional` with a list, you're doing nested lists.

This is why I always advocate to _always_ use `lib.optionals`, simply because it is explicit.

---

Resolved, thank you!